polymesh-docs/polymesh-private/development-environment/walkthrough-rest-api.md

```diff
@@ -715,259 +716,87 @@ Example response:
-              "assetId": "0x35f2d90772b65a565351124987913aea"
-            }
-          ],
-          "accountId": null,
-          "identityId": "0xbf84839e6bc8ad6fe969f560895663fdd2c175b40c9ab3ca63df9fd7010b64f8",
-          "party": "Sender",
-          "legId": 0
-        }
-      ]
-    }
-  }
-}
-```
-
-#### 5.6 Verify or decrypt sender proof using their confidential account to view the amount being sent
-
-```bash
-export RESPONSE_5_6=$(curl --silent --request 'POST' \
-  ''"$PP_REST_API"'/confidential-accounts/'"$RECEIVER_CONFIDENTIAL_ACCOUNT"'/receiver-verify' \
+export RESPONSE_5_5=$(curl --silent --request 'POST' \
+  ''"$PP_REST_API"'/confidential-transactions/'"$CONFIDENTIAL_TRANSACTION_ID"'/auditor-verify' \
```

Comment:
For the receiver verification should we not be using the `receiver-verify` method? It may be that the receiver verify method doesn't include fetching the proof from SubQuery so you used the auditor method instead but the example response you included in not sufficient for a receiver as it doesn't include an amount.